src/Command/Redis/SENTINEL.php

```diff
@@ -33,7 +33,10 @@ public function getId()
      */
     public function parseResponse($data)
     {
-        switch (strtolower($this->getArgument(0))) {
+        $argument = $this->getArgument(0);
+        $argument = is_null($argument) ? null : strtolower($argument);
+
+        switch ($argument) {
```

Comment:
What promoted this change? Is that coming from `v1.1` as well? Link?

Reply:
In `v1.1` the value is checked before it is sent to `strtolower` https://github.com/predis/predis/blob/d6da14daa81b63558ffd07ab5cc6946f207f4ac4/src/Command/ServerSentinel.php#L29-L37